.bundle/config

```diff
@@ -1,2 +1,2 @@
 ---
-BUNDLE_WITHOUT: "distribution"
+BUNDLE_WITH: "distribution"
```

Comment:
Do you know what this is from? I don't know anything about it but it's surprising to see it toggle to the opposite state.

Reply:
I don't remember changing this one, we could change it back

Reply:
If it's okay with you, I'm going to remove this change from the PR, out of caution.